heimdall/src/decompile/mod.rs

```diff
@@ -480,8 +480,8 @@ pub fn decompile(args: DecompilerArgs) {
             // resolve custom event signatures
             resolved_counter = 0;
             for (event_selector, (_, raw_event)) in analyzed_function.events.clone() {
-                decompilation_progress.set_message(format!("resolving event 0x{}", &event_selector.get(0..8).unwrap().to_string()));
-                let resolved_event_selectors = resolve_event_signature(&event_selector.get(0..8).unwrap().to_string());
+                decompilation_progress.set_message(format!("resolving event 0x{}", &event_selector.get(0..64).unwrap().to_string()));
```

Comment:
I would rather this still show only 4bytes, as it's just a progress bar.

Reply:
No problem -- just updated. Thanks for reviewing!